### PR TITLE
product card

### PR DESCRIPTION
**Pull request by zarva6596**
[DEMO LINK](https://zarva6596.github.io/layout_product-cards/)
[TEST REPORT LINK](https://zarva6596.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+      <div class="card__photo">
+        <img
+          src="images/imac.jpeg"
+          alt="photo"
+          class="card__img"
+        >
+      </div>
+      <div class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
```
Comment (vkryvytskyy) [inline src/index.html:24]: this should be a heading tag

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+      <div class="card__fit">
+        <div class="card__stars">
```
Comment (vkryvytskyy) [inline src/index.html:29]: card__stars should have modifier `--n`, which defines how many of stars should be active

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,53 @@
...
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__people">Отзывов: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__leyb">Цена:</div>
+        <div class="card__bigprice">69 999 грн</div>
```
Comment (vkryvytskyy) [inline src/index.html:40]: use spans for this elements

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.card__code {
+  height: 13px;
+  width: 101px;
+  font-style: normal;
+  font-weight: normal;
```
Comment (vkryvytskyy) [inline src/style.css:44]: this is default values for this property

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  width: 170px;
+  height: 31px;
+  line-height: 15px;
+  margin-bottom: 4px;
+  font-style: normal;
```
Comment (vkryvytskyy) [inline src/style.css:35]: this is default value

File: src/index.html (hunk 2 above)
Comment (vkryvytskyy) [reply src/index.html:29]: use your code from previous task

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 15px;
+  padding-left: 14px;
+  padding-right: 14px;
```
Comment (vkryvytskyy) [inline src/style.css:18]: combine into one rule

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+  background-position: center center;
+}
+
+.card__stars :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__people {
+  font-style: normal;
+  font-weight: normal;
```
Comment (vkryvytskyy) [inline src/style.css:76]: default values

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__leyb {
+  font-style: normal;
+  font-weight: normal;
```
Comment (vkryvytskyy) [inline src/style.css:89]: default values

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+  font-size: 11px;
+}
+
+.card__bigprice {
+  font-style: normal;
```
Comment (vkryvytskyy) [inline src/style.css:94]: default value

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+  padding: 12px 57px 11px 58px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.card__click:hover {
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  padding: 11px 56px 10px 57px;
```
Comment (vkryvytskyy) [inline src/style.css:120]: don't add border on hover, add it to default button state

Comment (zarva6596) [reply src/style.css:120]: готово

File: src/style.css (hunk 9 above)
Comment (zarva6596) [reply src/style.css:94]: готово

File: src/style.css (hunk 8 above)
Comment (zarva6596) [reply src/style.css:89]: готово

File: src/style.css (hunk 7 above)
Comment (zarva6596) [reply src/style.css:76]: готово

File: src/style.css (hunk 6 above)
Comment (zarva6596) [reply src/style.css:18]: готово

File: src/style.css (hunk 5 above)
Comment (zarva6596) [reply src/style.css:35]: готово

File: src/style.css (hunk 4 above)
Comment (zarva6596) [reply src/style.css:44]: готово

File: src/index.html (hunk 3 above)
Comment (zarva6596) [reply src/index.html:40]: готово

File: src/index.html (hunk 1 above)
Comment (zarva6596) [reply src/index.html:24]: готово

File: src/index.html (hunk 2 above)
Comment (zarva6596) [reply src/index.html:29]: готово

**Pull request closed without merging**